lib/emque/producing/message/message.rb

```diff
@@ -36,6 +36,14 @@ def read_raise_on_failure
           end
         end
 
+        def ignored_exceptions(ignored_exceptions)
+          @ignored_exceptions = Array(ignored_exceptions) + [InvalidMessageError, MessagesNotSentError]
```

Comment:
Should we automatically include `InvalidMessageError` and `MessageNotSentError` as part of the errors we ignore?